docs/cmdlet-example/visual-studio-simple-example.md

```diff
@@ -0,0 +1,125 @@
+# Building a C# Cmdlet with Visual Studio
+
+This document describes steps for building a C# Cmdlet with Visual Studio in 2 ways:
+
+* [Building a C# Cmdlet for PowerShell Core](#building-a-c-cmdlet-for-powershell-core-with-visual-studio)
+* [Building a C# Cmdlet for PowerShell Standard 3.0](#building-a-c-cmdlet-for-powershell-standard-30-with-visual-studio)
+
+## Building a C# Cmdlet for PowerShell Core with Visual Studio
+
+This demonstrates how to build your own C# cmdlet for PowerShell Core with Visual Studio.
+
+We will use the free [Visual Studio Community 2017](https://www.visualstudio.com/downloads).
+
+1. When installing Visual Studio 2017 select `.NET Core cross-platform development` under `Other Toolsets`
+  ![Step1](./Images/Step1.png)
+
+1. Create new C# project `SendGreeting` of type `Class Library (.NET Core)`
+  ![Step2](./Images/Step2.png)
+
+1. Now we need to setup PowerShell Core reference assemblies.
+  In `Solution Explorer` right click on project `Dependencies` and select `Manage NuGet Packages...`
+In the top-right corner of the package manager click on the small `Settings` sprocket icon that is to the right from `Package source` dropdown.
+By default, there will be only `nuget.org` package source in `Available package sources` list.
+Add another package source with name `powershell-core` and source `https://powershell.myget.org/F/powershell-core/api/v3/index.json`
+![Step3](./Images/Step3.png)
+In the package manager select new `powershell-core` in `Package source` dropdown, select `Browse` tab, type in `System.Management.Automation` in the search and select `Include prerelease`.
```

Comment:
This line is rendered to the same line of the image. Should be in a new line.

Reply:
Fixed.